.gitignore

```diff
@@ -2,3 +2,5 @@ venv
 .elasticbeanstalk/
 *.pyc
 __pycache__
+.idea/
```

Comment:
Can we keep these in the personal `.gitignore` since it's not really related to the project? @TheDoubleK has removed them from a few projects already.